### PR TITLE
nf-test rtgtools format

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/rtgtools/format/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                """
+                input[0] = [
+                    [id:'test'],
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_1.fastq.gz', checkIfExists: true),
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_2.fastq.gz', checkIfExists: true),
+                    []
+                ]
+                """
+            }
+        }
```
Comment (SPPearce) [inline modules/nf-core/rtgtools/format/tests/main.nf.test:66]: Is this a copy+paste error?

File: modules/nf-core/rtgtools/format/main.nf [hunk 2]
```diff
@@ -46,7 +46,6 @@ process RTGTOOLS_FORMAT {
     """
 
     stub:
-    def args = task.ext.args ?: ''
```
Comment (sateeshperi) [inline modules/nf-core/rtgtools/format/main.nf:49]: language server complaining abt this ?

File: modules/nf-core/rtgtools/format/tests/main.nf.test (hunk 1 above)
Comment (nvnieuwk) [reply modules/nf-core/rtgtools/format/tests/main.nf.test:66]: No it's a paired end fastq input :)

File: modules/nf-core/rtgtools/format/main.nf (hunk 2 above)
Comment (nvnieuwk) [reply modules/nf-core/rtgtools/format/main.nf:49]: yes :)

File: modules/nf-core/rtgtools/format/tests/main.nf.test [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
```
Comment (atrigila) [inline modules/nf-core/rtgtools/format/tests/main.nf.test:30]: I think capturing the versions is missing from this snapshot

Comment (nvnieuwk) [reply modules/nf-core/rtgtools/format/tests/main.nf.test:30]: Yes! thank you I missed that

**Review by atrigila — APPROVED** [review]
LGTM :)

**Pull request merged**